scripts/screener/screener.iefiles.ts

```diff
@@ -0,0 +1,82 @@
+const defaultExamples = [
```

Comment:
I don't really know how the screener setup works in v0, and wasn't ever really consolidated with the v7 screener setup. That would be ideal at some point but just to focus here;

It seems odd to me to have component names here in a general scripts file, far away from the components. If we add another component nobody will remember to go back to scripts/screener to add it to this file.

We have the screener setup in a storybook projects in apps/vr-tests. It's pretty simple to have running and easy to know where to look for which tests will run. Not applicable to this PR specifically but I think it'd be great to just combine the projects, OR, keeps separate vr-tests app projects for multiple runs if we get parallel benefits but at least know where to look for what tests run.




We have a project

Reply:
I agree that keeping the list far from the components itself is not ideal, what we were trying to achieve was to not run the IE11 for all components examples since it takes long time. So the idea would be to have a specific list. Another way to achieve the same that I can see is if for the examples that should be tested on ie we would add something in the file name like `PopupExample.ie.shorthand.tsx` this way we can match and include only these specific files to run on IE11 screener.